servicetalk-loadbalancer-experimental-provider/build.gradle

```diff
@@ -17,18 +17,10 @@
 apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
 
 dependencies {
-  implementation platform(project(":servicetalk-dependencies"))
-  testImplementation enforcedPlatform("org.junit:junit-bom:$junit5Version")
-
-  api project(":servicetalk-client-api")
-  api project(":servicetalk-concurrent-api")
```

Comment:
We still need to keep `api project(":servicetalk-http-api")` bcz your provider is a public class that depends on `http-api`, and it's not final.